playwright-e2e/lib/component/dsm/table/table.ts

```diff
@@ -0,0 +1,51 @@
+import { Locator, Page } from '@playwright/test';
+import ParticipantPage from 'pages/dsm/participant-page';
+
+export class Table {
```

Comment:
Is this new class created for Participant table only? If yes, we should rename it. If no, other types of tables cannot use this.
Also, could you add jsdoc for all functions?

Reply:
This table can be used for the Tissue List page as well, there is a similar table. 
I also thought of renaming it and actually, I did it locally. Let me push.